### PR TITLE
Notes support

### PR DESCRIPTION
**Pull request by charger**

### CODE REVIEW

File: lib/netsuite/records/note.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module NetSuite
+  module Records
+    class Note
+      OUTGOING = '_outgoing'
+      INCOMING = '_incoming'
```
Comment (iloveitaly) [inline lib/netsuite/records/note.rb]: @charger what's the idea behind these constants? Is this something that the gem user will use?

**Comment by iloveitaly** [comment]
@charger Thanks for this!

Could you remove the wsdl from the PR? I've stopped including the wsdls in the gem since the URLs in the wsdls can change depending on the data center URL that is used.

File: lib/netsuite/records/note.rb (hunk 1 above)
Comment (charger) [inline lib/netsuite/records/note.rb]: @iloveitaly Yes, gem user can use these constants. Maybe this is better than use magic strings.

Comment (iloveitaly) [inline lib/netsuite/records/note.rb]: I'm definitely a fan of avoiding magic strings. However, right now we don't include magic strings in the gem for other aspects of the API. It's best to omit these from the gem for now until we create a generic solution for handling magic strings.

Comment (charger) [inline lib/netsuite/records/note.rb]: done

**Comment by iloveitaly** [comment]
Looks great @charger! Thanks for the PR, and for making those tweaks.

This is merged with 34d42a4...0b48a12

**Pull request closed without merging**